unit_testing/test_reports.py

```diff
@@ -8,31 +8,50 @@
 from spinalcordtoolbox.image import Image
 from spinalcordtoolbox.reports.slice import Sagittal
 
+@pytest.fixture(scope="module")
+def im_base(path_in='t2/t2.nii.gz'):
+    # Base anatomical image
+    yield Image(path_in)
 
-def labeled_data_test_params(path_in='sct_testing_data/t2/t2.nii.gz',
-                             path_seg='sct_testing_data/t2/labels.nii.gz'):
-    """Generate image/label pairs for various test cases of
-    test_sagittal_slice_get_center_spit."""
-    im_in = Image(path_in)            # Base anatomical image
-    im_seg_labeled = Image(path_seg)  # Base labeled segmentation
-    assert np.count_nonzero(im_seg_labeled.data) >= 2, "Labeled segmentation image has fewer than 2 labels"
+@pytest.fixture(scope="module")
+def im_seg_labeled(path_seg='t2/labels.nii.gz'):
```

Comment:
Fix for failing CI: (2/2)
```suggestion
def im_seg_labeled(path_seg=sct_test_path('t2', 'labels.nii.gz')):
```

Reply:
Oh yeah this used to say `sct_testing_data/` instead of calling `sct_test_path`.

I guess there's no harm fixing this here instead of #3125! Let's put it in :)